_tarsnap

```diff
@@ -0,0 +1,163 @@
+#compdef tarsnap
+
+# zsh completion function for tarsnap 1.0.37
+
+# Set the following variable to the path of a file containing the output from "tarsnap --list-archives" 
+# i.e. one archive name per line. If left blank then archive names will not be completed.
+local archivefile=
```

Comment:
A little nitpicky, but could this variable be renamed to something like `archivelist_file` ?  (or something similar, if underscores aren't allowed in zsh)
My first impression was that this was supposed to be a list of archives; I had to read the comment twice before I understood that it was supposed to point to a text file which contains a list of archive names.